backend/frontend/views.py

```diff
@@ -295,6 +298,75 @@ def action_start_card(request, card_id):
     )
 
 
+def user_can_request_review(logged_in_user):
+    request = get_current_request()
+    card_id = request.resolver_match.kwargs.get("card_id")
+
+    card = get_object_or_404(AgileCard, pk=card_id)
+    card_assignees = card.assignees.all()
+
+    if logged_in_user in card_assignees:
```

Comment:
This functionality seems to be repeated in AgileCard.request_user_can_request_review